src/chainparams.cpp

```diff
@@ -85,7 +85,9 @@ class CMainParams : public CChainParams {
         consensus.BIP34Height = 227931;
         consensus.BIP34Hash = uint256S("0x000000000000024b89b42a942fe0d9fea3bb44ab7bd1b19115dd6a759c0808b8");
         consensus.powLimit = uint256S("00000000ffffffffffffffffffffffffffffffffffffffffffffffffffffffff");
+        consensus.aux_pow_limit = uint256S("7fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff");
         consensus.nPowTargetSpacing = 10 * 60;
+        consensus.aux_pow_target_spacing = 12 * 60;
```

Comment:
It probably does need to be explained, although I'm not sure that's the place to do it. Maybe I should add a AUXPOW.md file in the docs directory?

The reason, and tell me if this short explanation makes sense, is that the expected block interval post-activation is lengthened to 12 minutes per block (720 seconds) instead of the usual 10 min (600 s), since the auxiliary difficulty will keep rising until blocks are coming 12 minutes apart.  The reason for that is to achieve the transition of difficulty between the two proofs of work. The auxiliary difficulty will quickly rise until solving the auxpow consumes >2 min of time on average, which makes the overall expected block time 12 minutes. The native difficulty is already dropping at that point since it is targeting 10 minutes blocks--from its perspective, 20% of the hash power just disappeared. When the native difficulty drops, the auxiliary difficulty rises to compensate. This continues until the native difficulty is its minimum value, and the full 12 minutes per block is spent in the auxiliary proof-of-work.

Having explained this, I realize now that this interferes with the 5% demurrage rate. Having a 12 minute block time means an effective annual demurrage rate of 4.25% instead of 5%. Permanently changing that is unacceptable... I should probably look into restoring 10 minute block times once the native difficulty reaches 1.

Reply:
Wait, so the 12 minute block rate will essentially result in a natural 'drop block difficulty cause we're > 10 mins!" and the drop will result in aux pow going 'raise block difficulty cause we're < 12 mins!".

That's pretty clever!